spec/book_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rspec'
+require_relative '../classes/book'
+
+describe Book do
+  context 'Testing book class' do
+    it 'A book should have a title and author' do
+      book = Book.new('Title', 'Author')
+      title = book.title
+      author = book.author
+      book_title_author = [title, author]
+
+      expect(book_title_author).to eq %w[Title Author]
+    end
+  end
```

Comment:
- [ ] All class methods should be tested. Kindly add test for your `add_rental` method as contained in the **Book** class

Reply:
Test added for add_rental